### PR TITLE
feat: add test coverage for webapp.

### PR DESCRIPTION
**Pull request by cbernat**
### Add test coverage

This PR is based [on this other PR.](https://github.com/FromDoppler/unlayer-editor/pull/31)
To force an amount of testing per feature, this seems to be a good idea.
`yarn test:ci` gives this result

```
-----------------------------------------------------------------------|---------|----------|---------|---------|--------------------------------------------
File                                                                   | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
-----------------------------------------------------------------------|---------|----------|---------|---------|--------------------------------------------
All files                                                              |   79.45 |    69.02 |   79.24 |   79.43 | 
 src                                                                   |   84.47 |    79.65 |   89.29 |   84.21 |                                            
  App.js                                                               |      80 |    54.55 |      80 |   82.61 | 81-86,106
  utils.ts                                                             |   79.59 |    75.47 |   89.47 |   78.02 | 97,131-150,155,199,240-247
  validations.ts                                                       |     100 |      100 |     100 |     100 |                                            
 src/components                                                        |   72.55 |       50 |   68.75 |      72 |                                            
  PrivateRoute.js                                                      |     100 |    36.36 |     100 |     100 | 36-46                                      
  PublicRouteWithLegacyFallback.js                                     |   82.14 |       75 |   66.67 |   81.48 | 47-54,77,105-106
  RedirectToExternalUrl.js                                             |     100 |      100 |     100 |     100 | 
  RedirectToLogin.js                                                   |     100 |      100 |     100 |     100 |                                            
  RedirectWithQuery.js                                                 |      10 |        0 |       0 |      10 | 5-8,12-27                                  
  SafeRedirect.js                                                      |     100 |      100 |     100 |     100 |                                            
 src/components/ChangePlan                                             |   88.75 |    96.77 |      75 |   89.04 | 
  Card.js                                                              |     100 |      100 |     100 |     100 | 
  ChangePlan.js                                                        |   86.15 |    96.43 |      70 |   86.44 | 25-73,119                                  
  ChangePlan.styles.js                                                 |     100 |      100 |     100 |     100 |                                            
 src/components/ChangePlan/Checkout                                    |   31.58 |        0 |   22.22 |   29.73 | 
  Checkout.js                                                          |   36.67 |        0 |      25 |   34.48 | 27-44,75-107
  CheckoutSummary.js                                                   |    12.5 |        0 |       0 |    12.5 | 7-17
 src/components/ChangePlan/Checkout/BillingInformation                 |      74 |    73.81 |   61.54 |   73.47 |                                            
  BillingInformation.js                                                |      74 |    73.81 |   61.54 |   73.47 | 103-110,116-119,158-238
 src/components/ChangePlan/Checkout/ContactInformation                 |   88.89 |       50 |   84.62 |   88.64 |                                            
  ContactInformation.js                                                |   88.89 |       50 |   84.62 |   88.64 | 97-100,190
 src/components/ChangePlan/Checkout/Discounts                          |   68.18 |    70.59 |    62.5 |   66.67 | 
  Discounts.js                                                         |      65 |    70.59 |    62.5 |   63.16 | 26,31,36-39,70-71                          
  Discounts.styles.js                                                  |     100 |      100 |     100 |     100 |                                            
 src/components/ChangePlan/Checkout/PaymentMethod                      |   93.55 |    84.21 |    87.5 |   93.33 |                                            
  PaymentMethod.js                                                     |   93.55 |    84.21 |    87.5 |   93.33 | 49,104                                     
 src/components/ChangePlan/Checkout/Step                               |     100 |      100 |     100 |     100 | 
  Step.js                                                              |     100 |      100 |     100 |     100 | 
 src/components/ChangePlan/PlanCalculator                              |    90.2 |       70 |   84.62 |    89.8 |                                            
  PlanCalculator.js                                                    |    89.8 |       70 |   84.62 |   89.36 | 41,54-61,112,166                           
  PlanCalculator.styles.js                                             |     100 |      100 |     100 |     100 |                                            
 src/components/ChangePlan/PlanCalculator/BannerUpgrade                |      85 |      100 |      40 |      85 |                                            
  BannerUpgrade.js                                                     |      85 |      100 |      40 |      85 | 63,80-81
 src/components/ChangePlan/PlanCalculator/Discounts                    |   90.91 |    92.31 |    87.5 |   90.48 |                                            
  Discounts.js                                                         |      90 |    92.31 |    87.5 |   89.47 | 18,68                                      
  Discounts.styles.js                                                  |     100 |      100 |     100 |     100 |                                            
 src/components/ChangePlan/PlanCalculator/NavigatorTabs                |      90 |    83.33 |     100 |   88.89 | 
  NavigatorTabs.js                                                     |      90 |    83.33 |     100 |   88.89 | 28
 src/components/ChangePlan/PlanCalculator/PlanAgreement                |   92.86 |       90 |     100 |   92.31 | 
  PlanAgreement.js                                                     |   92.86 |       90 |     100 |   92.31 | 26
 src/components/ChangePlan/PlanCalculator/PlanPrice                    |     100 |      100 |     100 |     100 | 
  PlanPrice.js                                                         |     100 |      100 |     100 |     100 | 
 src/components/ChangePlan/PlanCalculator/PlanPricePerMonth            |     100 |      100 |     100 |     100 | 
  PlanPricePerMonth.js                                                 |     100 |      100 |     100 |     100 |                                            
 src/components/ChangePlan/PlanCalculator/PlanPriceWithoutDiscounts    |     100 |      100 |     100 |     100 | 
  PlanPriceWithoutDiscounts.js                                         |     100 |      100 |     100 |     100 | 
 src/components/CloudTags                                              |     100 |      100 |     100 |     100 | 
  index.js                                                             |     100 |      100 |     100 |     100 | 
 src/components/ContactPolicy                                          |    90.2 |    94.12 |   78.26 |    90.1 |                                            
  ContactPolicy.js                                                     |    90.2 |    94.12 |   78.26 |    90.1 | 57-58,94-95,159-160,324-339                
 src/components/Dashboard                                              |     100 |      100 |     100 |     100 |                                            
  Dashboard.js                                                         |     100 |      100 |     100 |     100 |                                            
 src/components/DopplerPlus                                            |   98.72 |    71.43 |   95.83 |   98.67 | 
  AgenciesForm.js                                                      |     100 |       75 |     100 |     100 | 79
  ExclusiveForm.js                                                     |   97.06 |    66.67 |   91.67 |   96.97 | 128
  ExclusiveForm.styles.js                                              |     100 |      100 |     100 |     100 |                                            
  UpgradeSuggestionForm.js                                             |     100 |       75 |     100 |     100 | 49                                         
 src/components/FAQ                                                    |     100 |      100 |     100 |     100 |                                            
  constants.js                                                         |     100 |      100 |     100 |     100 | 
  index.js                                                             |     100 |      100 |     100 |     100 | 
 src/components/FAQ/QAItem                                             |     100 |      100 |     100 |     100 |                                            
  index.js                                                             |     100 |      100 |     100 |     100 |                                            
 src/components/Footer                                                 |     100 |      100 |     100 |     100 |                                            
  Footer.js                                                            |     100 |      100 |     100 |     100 | 
 src/components/ForgotPassword                                         |   86.67 |       70 |     100 |   86.21 | 
  ForgotPassword.js                                                    |   86.21 |       70 |     100 |   85.71 | 43,61-62,68
  ForgotPassword.styles.js                                             |     100 |      100 |     100 |     100 | 
 src/components/Header                                                 |      80 |    67.86 |    62.5 |      80 |                                            
  Header.js                                                            |   73.33 |       50 |      50 |   73.33 | 10-11,23-33
  Notifications.js                                                     |     100 |      100 |     100 |     100 | 
 src/components/Header/HeaderMessages                                  |      50 |    81.82 |      20 |   57.14 | 
  HeaderMessages.js                                                    |      50 |    81.82 |      20 |   57.14 | 26-37
 src/components/Header/HeaderNav                                       |      50 |        0 |   33.33 |      50 | 
  HeaderNav.js                                                         |      50 |        0 |   33.33 |      50 | 9-16
 src/components/Header/HeaderUserMenu                                  |   64.29 |       75 |    37.5 |   66.67 | 
  HeaderUserMenu.js                                                    |   64.29 |       75 |    37.5 |   66.67 | 52-63,163-166                              
 src/components/Integrations/BigQuery                                  |   68.57 |       50 |   83.33 |   67.65 | 
  AuthorizationPage.js                                                 |   68.57 |       50 |   83.33 |   67.65 | 26,35-49                                   
 src/components/Integrations/BigQuery/ControlPanel                     |     100 |      100 |     100 |     100 | 
  AuthorizationForm.js                                                 |     100 |      100 |     100 |     100 |                                            
 src/components/Integrations/Shopify                                   |   97.22 |    85.71 |      90 |   97.06 | 
  Shopify.js                                                           |   96.97 |    85.71 |   88.89 |   96.88 | 234
  Shopify.styles.js                                                    |     100 |      100 |     100 |     100 |                                            
 src/components/InvoicesList                                           |     100 |    93.75 |     100 |     100 |                                            
  InvoicesList.js                                                      |     100 |    93.75 |     100 |     100 | 142                                        
 src/components/Loading                                                |     100 |      100 |     100 |     100 | 
  Loading.js                                                           |     100 |      100 |     100 |     100 |                                            
 src/components/Login                                                  |      40 |    27.78 |   55.56 |   39.13 | 
  Login.js                                                             |   44.58 |    29.07 |   66.67 |   43.21 | 49-50,58-66,80-93,128,168-258,263,267-268  
  LoginErrorAccountNotValidated.js                                     |    8.33 |        0 |       0 |    9.09 | 13-29                                      
 src/components/Modal                                                  |     100 |      100 |     100 |     100 | 
  Modal.js                                                             |     100 |      100 |     100 |     100 |                                            
 src/components/NewFeatures                                            |      50 |      100 |       0 |      50 | 
  NewFeatures.js                                                       |      50 |      100 |       0 |      50 | 7
 src/components/Offline                                                |   85.71 |      100 |       0 |   85.71 |                                            
  Offline.js                                                           |      50 |      100 |       0 |      50 | 6                                          
  Offline.styles.js                                                    |     100 |      100 |     100 |     100 |                                            
 src/components/Reports                                                |      75 |    73.08 |   58.33 |   82.14 | 
  Reports.js                                                           |      75 |    73.08 |   58.33 |   82.14 | 41-42,46-47,65
 src/components/Reports/GDPRPermissions                                |   94.92 |     87.5 |   94.12 |   94.74 |                                            
  PermissionExpandableRow.js                                           |     100 |    92.86 |     100 |     100 | 9-11                                       
  SubscriberGdpr.styles.js                                             |     100 |      100 |     100 |     100 |                                            
  SubscriberGdprPermissions.js                                         |   88.46 |       75 |   88.89 |      88 | 21,36,47
 src/components/Reports/MasterSubscriber                               |   52.17 |    22.22 |   57.14 |      50 | 
  MasterSubscriber.js                                                  |   52.17 |    22.22 |   57.14 |      50 | 22-27,31-34,86-88                          
 src/components/Reports/MasterSubscriber/MasterSubscriberCurrentSearch |   81.82 |       75 |   77.78 |   81.82 |                                            
  MasterSubscriberCurrentSearch.js                                     |   81.82 |       75 |   77.78 |   81.82 | 30-37                                      
 src/components/Reports/ReportsBox                                     |     100 |      100 |     100 |     100 | 
  ReportsBox.js                                                        |     100 |      100 |     100 |     100 | 
 src/components/Reports/ReportsDailyVisits                             |   48.15 |    33.33 |   42.86 |   48.15 |                                            
  ReportsDailyVisits.js                                                |   48.15 |    33.33 |   42.86 |   48.15 | 22-74,103                                  
 src/components/Reports/ReportsFilters                                 |   81.82 |      100 |   77.78 |   81.82 |                                            
  ReportsFilters.js                                                    |      80 |      100 |   77.78 |      80 | 32,81                                      
  ReportsFilters.styles.js                                             |     100 |      100 |     100 |     100 |                                            
 src/components/Reports/ReportsHoursVisits                             |     100 |     87.5 |     100 |     100 | 
  ReportsHoursVisits.js                                                |     100 |       85 |     100 |     100 | 52-58,140
  ReportsHoursVisits.styles.js                                         |     100 |      100 |     100 |     100 |                                            
 src/components/Reports/ReportsPageRanking                             |   90.63 |       80 |      75 |   93.55 |                                            
  ReportsPageRanking.js                                                |   91.67 |       80 |   85.71 |   91.67 | 31,56                                      
  ReportsPageRanking.styles.js                                         |    87.5 |      100 |       0 |     100 | 
 src/components/Reports/ReportsPartialsCampaigns                       |     100 |    85.71 |     100 |     100 | 
  ReportsPartialsCampaigns.js                                          |     100 |    85.71 |     100 |     100 | 215-247                                    
  ReportsPartialsCampaigns.styles.js                                   |     100 |      100 |     100 |     100 |                                            
 src/components/Reports/ReportsTrafficSources                          |     100 |       75 |     100 |     100 |                                            
  ReportsTrafficSources.js                                             |     100 |       75 |     100 |     100 | 8,62-89
  ReportsTrafficSources.styles.js                                      |     100 |       75 |     100 |     100 | 122
 src/components/Reports/SubscriberSentCampaigns                        |   89.47 |    76.19 |     100 |   89.19 |                                            
  SubscriberSentCampaigns.js                                           |   89.47 |    76.19 |     100 |   89.19 | 29,50,60,76                                
 src/components/Reports/Subscribers                                    |   85.71 |    61.54 |   85.71 |   85.29 |                                            
  Subscribers.js                                                       |     100 |      100 |     100 |     100 | 
  SubscribersLegacyUrlRedirect.js                                      |   16.67 |        0 |       0 |   16.67 | 9-16                                       
 src/components/Signup                                                 |   69.79 |    55.36 |   72.22 |   70.33 | 
  Signup.js                                                            |   71.43 |    56.25 |   78.57 |   71.62 | 106,111,136,168-199
  Signup.styles.js                                                     |     100 |      100 |     100 |     100 |                                            
  SignupConfirmation.js                                                |   56.25 |       50 |      50 |   57.14 | 22,25-30
  SignupConfirmation.styles.js                                         |     100 |      100 |     100 |     100 | 
 src/components/SiteTrackingRequired                                   |   89.47 |    81.82 |     100 |   88.89 | 
  SiteTrackingRequired.js                                              |   89.47 |    81.82 |     100 |   88.89 | 34,42                                      
 src/components/SubscriberListSelector                                 |   98.15 |    97.06 |   94.12 |   98.04 |                                            
  SubscriberListSelector.js                                            |   98.15 |    97.06 |   94.12 |   98.04 | 160
 src/components/TooltipContainer                                       |     100 |      100 |     100 |     100 |                                            
  TooltipContainer.js                                                  |     100 |      100 |     100 |     100 | 
  TooltipContainer.styles.js                                           |     100 |      100 |     100 |     100 | 
 src/components/UpgradePlanForm                                        |    62.5 |    55.17 |   55.56 |    61.7 | 
  ConfirmationMessage.js                                               |   83.33 |      100 |   66.67 |      80 | 24                                         
  UpgradePlanForm.js                                                   |   59.52 |    51.85 |   53.33 |   59.52 | 23-25,62-81,87-91,159                      
 src/components/form-helpers                                           |      80 |    76.32 |   76.79 |   82.39 | 
  captcha-utils.tsx                                                    |   48.39 |    14.29 |    37.5 |      50 | 41,48-70                                   
  form-helpers.js                                                      |   87.05 |    82.61 |   83.33 |   89.92 | 59,112-123,154,304,308,313-314,363-364,441
 src/components/form-helpers/CloudTagCompoundField                     |     100 |       50 |     100 |     100 |                                            
  index.js                                                             |     100 |       50 |     100 |     100 | 26
 src/components/form-helpers/CloudTagField                             |     100 |       90 |     100 |     100 | 
  index.js                                                             |     100 |       90 |     100 |     100 | 57                                         
 src/components/shared/Breadcrumb                                      |     100 |      100 |     100 |     100 |                                            
  Breadcrumb.js                                                        |     100 |      100 |     100 |     100 |                                            
 src/components/shared/C3Chart                                         |   17.65 |    33.33 |       0 |   17.65 |                                            
  C3Chart.js                                                           |    6.67 |        0 |       0 |    6.67 | 7-30                                       
  C3Chart.styles.js                                                    |     100 |      100 |     100 |     100 |                                            
  SVGPathElement.patch.js                                              |     100 |      100 |     100 |     100 | 
 src/components/shared/ConfirmationBox                                 |     100 |      100 |     100 |     100 | 
  ConfirmationBox.js                                                   |     100 |      100 |     100 |     100 |                                            
  ConfirmationBox.styles.js                                            |     100 |      100 |     100 |     100 |                                            
 src/components/shared/FormattedDateRangeText                          |     100 |      100 |     100 |     100 |                                            
  FormattedDateRangeText.js                                            |     100 |      100 |     100 |     100 | 
 src/components/shared/HeaderSection                                   |     100 |      100 |     100 |     100 | 
  HeaderSection.js                                                     |     100 |      100 |     100 |     100 |                                            
 src/components/shared/LanguageSelector                                |   81.82 |     87.5 |     100 |   81.82 |                                            
  LanguageSelector.js                                                  |   81.82 |     87.5 |     100 |   81.82 | 24-25
 src/components/shared/Pagination                                      |   91.67 |       80 |   83.33 |    91.3 |                                            
  Pagination.js                                                        |   91.67 |       80 |   83.33 |    91.3 | 66,88                                      
 src/components/shared/Promotional                                     |     100 |      100 |     100 |     100 |                                            
  Promotional.js                                                       |     100 |      100 |     100 |     100 |                                            
  Promotional.styles.js                                                |     100 |      100 |     100 |     100 | 
 src/components/shared/Promotions                                      |     100 |      100 |     100 |     100 | 
  Promotions.js                                                        |     100 |      100 |     100 |     100 |                                            
 src/components/shared/ShowLikeFlash                                   |      90 |       75 |   66.67 |     100 |                                            
  ShowLikeFlash.js                                                     |      90 |       75 |   66.67 |     100 | 16                                         
 src/components/shared/Slider                                          |     100 |      100 |     100 |     100 | 
  Slider.js                                                            |     100 |      100 |     100 |     100 | 
 src/components/shared/StarsScore                                      |     100 |      100 |     100 |     100 | 
  StarsScore.js                                                        |     100 |      100 |     100 |     100 |                                            
 src/components/shared/SubscriberInfo                                  |     100 |       50 |     100 |     100 | 
  SubscriberInfo.js                                                    |     100 |       50 |     100 |     100 | 19-28
 src/components/shared/Tabs                                            |     100 |      100 |     100 |     100 | 
  Tabs.js                                                              |     100 |      100 |     100 |     100 | 
 src/components/styles                                                 |     100 |      100 |     100 |     100 |                                            
  animations.js                                                        |     100 |      100 |     100 |     100 | 
  colors.js                                                            |     100 |      100 |     100 |     100 |                                            
  fonts.js                                                             |     100 |      100 |     100 |     100 | 
  messages.js                                                          |     100 |      100 |     100 |     100 | 
  spacings.js                                                          |     100 |      100 |     100 |     100 |                                            
 src/hooks/useCloudTags                                                |     100 |      100 |     100 |     100 |                                            
  index.js                                                             |     100 |      100 |     100 |     100 |                                            
 src/hooks/useInfinitePaged                                            |   96.67 |       75 |     100 |     100 | 
  index.js                                                             |   96.67 |       75 |     100 |     100 | 28                                         
 src/hooks/useTimeout                                                  |   92.31 |       75 |     100 |      90 | 
  index.js                                                             |   92.31 |       75 |     100 |      90 | 14                                         
 src/i18n                                                              |     100 |      100 |     100 |     100 | 
  DopplerIntlProvider.double-with-ids-as-values.js                     |     100 |      100 |     100 |     100 | 
  DopplerIntlProvider.js                                               |     100 |      100 |     100 |     100 | 
  FormattedMessageMarkdown.js                                          |     100 |      100 |     100 |     100 | 
  en.js                                                                |     100 |      100 |     100 |     100 | 
  es.js                                                                |     100 |      100 |     100 |     100 |                                            
  utils.js                                                             |     100 |      100 |     100 |     100 | 
 src/services                                                          |   68.42 |    56.38 |   77.27 |   68.03 | 
  app-session.ts                                                       |     100 |      100 |     100 |     100 |                                            
  big-query-client.ts                                                  |   30.43 |        0 |      50 |   30.43 | 38-80                                      
  datahub-client.ts                                                    |   88.89 |    78.95 |   93.33 |   90.24 | 135,193,282,363                            
  doppler-account-plans-api-client.ts                                  |   78.26 |    66.67 |     100 |   77.27 | 54,65-71                                   
  doppler-api-client.ts                                                |   87.25 |    70.73 |      96 |   87.76 | 268-272,283-284,298-302,313,582
  doppler-billing-api-client.ts                                        |    91.3 |    88.89 |     100 |   90.91 | 63,120                                     
  doppler-billing-user-api-client.ts                                   |    87.1 |    65.38 |     100 |    87.1 | 76,143-146,167
  doppler-contact-policy-api-client.ts                                 |    87.5 |    72.73 |     100 |   86.96 | 58,101,121                                 
  doppler-legacy-client.ts                                             |   18.94 |     31.1 |   24.24 |   19.23 | 189,445-502,568-577,603-609,628-946       
  doppler-sites-client.ts                                              |      50 |    33.33 |     100 |      50 | 43-58
  doppler-user-api-client.ts                                           |   86.21 |     87.5 |     100 |   86.21 | 108-111,132,148                            
  experimental-features.js                                             |     100 |      100 |     100 |     100 |                                            
  http-base-client.ts                                                  |   85.71 |       75 |     100 |   85.71 | 39                                         
  ipinfo-client.ts                                                     |     100 |      100 |     100 |     100 | 
  manual-status-client.ts                                              |   28.57 |        0 |      50 |   28.57 | 24-35
  origin-management.tsx                                                |     100 |    94.12 |     100 |     100 | 21                                         
  plan-service.ts                                                      |   82.24 |     67.8 |   85.71 |      82 | 107,139,197-198,212-219,285-309,331,356    
  pure-di.tsx                                                          |   86.67 |      100 |   85.71 |   84.91 | 141,164,183,195,266,321,332,344            
  session-manager.ts                                                   |   22.86 |        0 |    12.5 |   22.86 | 46-112
  shopify-client.ts                                                    |   93.75 |    93.33 |     100 |   93.75 | 54
  static-data-client.ts                                                |   61.54 |    33.33 |   44.44 |   66.67 | 41,47,58-71
  utm-cookies-manager.js                                               |     100 |      100 |     100 |     100 | 
 src/services/test-utils                                               |     100 |      100 |     100 |     100 | 
  local-storage-double.js                                              |     100 |      100 |     100 |     100 | 
-----------------------------------------------------------------------|---------|----------|---------|---------|--------------------------------------------
```

### CODE REVIEW

**Comment by cbernat** [comment]
@andresmoschini @paangaflo any comments?

File: package.json [hunk 1]
```diff
@@ -59,12 +59,14 @@
...
     "build": "react-scripts build",
-    "verify": "yarn prettier-check && yarn eclint && yarn test:ci && yarn stylelint",
+    "verify": "yarn prettier-check && yarn eclint && yarn coverage:ci && yarn stylelint",
     "build:production": "env-cmd -f .env.production yarn build",
     "build:qa": "env-cmd -f .env.qa yarn build",
     "build:int": "env-cmd -f .env.int yarn build",
     "build:development": "env-cmd -f .env.development yarn build",
     "build:demo": "env-cmd -f .env.demo yarn build",
+    "coverage:local": "react-scripts test --coverage --watchAll --env=jest-environment-jsdom-sixteen --testTimeout=120000",
+    "coverage:ci": "cross-env CI=true react-scripts test --coverage --env=jest-environment-jsdom-sixteen --testTimeout=120000",
```
Comment (andresmoschini) [inline package.json:69]: Creo que no es necesario tener 2

Comment (andresmoschini) [reply package.json:69]: Ah, perdón, si. 

En nuestro caso no es necesario porque la variable de entorno la seteaos en Docker.

**Comment by cbernat** [comment]
When this is merged, to run tests you will need to run `yarn coverage:local` instead of `yarn test` @silsanchez8 @jcamposmk @jhoffman-ms @kapoc3 @GusBaamonde

**Comment by paangaflo** [comment]
Cool!.. Buen trabajo @cbernat

**Comment by andresmoschini** [comment]
> When this is merged, to run tests you will need to run `yarn coverage:local` instead of `yarn test` @silsanchez8 @jcamposmk @jhoffman-ms @kapoc3 @GusBaamonde

What do you think about update `test` command in place of adding a new `coverage` command?

File: package.json (hunk 1 above)
Comment (cbernat) [reply package.json:69]: The local command keeps watching, I think it`s useful when coding. 🤔 Am I missing something?

Comment (cbernat) [reply package.json:69]: If you are talking about renaming these into tests, it may be better to avoid misunderstandings 😅

File: package.json [hunk 2]
```diff
@@ -59,12 +59,14 @@
...
     "test:ci": "cross-env CI=true react-scripts test --env=jest-environment-jsdom-sixteen --testTimeout=120000",
     "test:related": "react-scripts test --env=jest-environment-jsdom-sixteen --testTimeout=120000 -b --findRelatedTests",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen --watchAll=true --testTimeout=120000",
```
Comment (andresmoschini) [inline package.json:72]:
```suggestion
    "test": "react-scripts test --coverage --watchAll --env=jest-environment-jsdom-sixteen --testTimeout=120000",x
    "test:ci": "cross-env CI=true react-scripts test --coverage --env=jest-environment-jsdom-sixteen --testTimeout=120000",
    "test:related": "react-scripts test --env=jest-environment-jsdom-sixteen --testTimeout=120000 -b --findRelatedTests",
```

File: package.json (hunk 1 above)
Comment (andresmoschini) [reply package.json:69]: Right, like this: https://github.com/FromDoppler/doppler-webapp/pull/1593/files#r699546612

**Comment by maurocardinali** [comment]
Great job Ceci! 🚀

**Comment by silsanchez8** [comment]
genial Ceci!! Entonces seguimos corriendo `yarn test` 😉  ?

**Pull request merged**